api/src/main/resources/liquibase.xml

```diff
@@ -306,4 +306,27 @@
             <column name="uuid" value="8b5caeea-2c3b-11e8-b467-0ed5f89f718b" />
         </insert>
     </changeSet>
+
+	<changeSet author="rmakoba" id="ugandaemr-04052020-iss-9">
```

Comment:
@reagan-meant You need to add a check that the table does not exist

Reply:
Done